src/open_inwoner/scss/components/Header/AnchorMenu.scss

```diff
@@ -82,6 +82,30 @@
     }
   }
 
+  // nested list
+  &__sublist {
+    list-style-type: '- ';
```

Comment:
I would like to show this to a designer actually - because I think this should be 'list-style-type:none; so it looks cleaner and ore in line with the rest of the website, but i'm really not sure about this one. So perhaps leave it as is.

Reply:
After playing around a bit with different styles, I think you're right. Changed it accordingly.